tests/test_integration.py

```diff
@@ -9,6 +10,9 @@
 import stripe
 import pytest
 
+if platform.python_implementation() == "PyPy":
+    pytest.skip("skip integration tests with PyPy", allow_module_level=True)
```

Comment:
I opted to skip integration tests with PyPy entirely to avoid more transient errors.